src/js/challenge-card.js

```diff
@@ -59,9 +59,19 @@ export function createChallengeCard(challenge) {
     description.textContent = challenge.description;
     item.appendChild(description);
 
+    const apiTitle = document.createElement("input");
+    apiTitle.setAttribute("hidden", "true")
+    apiTitle.setAttribute("value", challenge.title)
+
+    const apiId = document.createElement("input");
+    apiId.setAttribute("hidden", "true")
+    apiId.setAttribute("value", challenge.id)
+
     const button = document.createElement('button');
     button.classList.add('button', 'primary');
     button.textContent = buttonText;
+    button.append(apiTitle);
+    button.append(apiId);
```

Comment:
hmm kom på att funktionen "createModal" tar in hela challenge objektet. Kanske bättre att spara hela challenge objektet i knappen istället för id och titel separat? Annars kommer det bli problem när modalen skapas tror jag.

Reply:
Jag kan ändra så att hela objektet sparas i ett input, det blir dock som en string så du får köra nån JSON.parse() eller liknande i createModal funktionen